frontend/src/tests/components/Forms/__snapshots__/PermitAmendmentForm.spec.js.snap

```diff
@@ -10,58 +10,6 @@ exports[`PermitAmendmentForm renders properly 1`] = `
   <Row
     gutter={48}
   >
-    <Col
-      md={12}
-      sm={24}
-    >
-      <FormItem
-        hasFeedback={false}
-      >
-        <Connect(PartySelectField)
-          allowAddingParties={true}
```

Comment:
Just pointing out that the snapshot test does not render the component with isMajorMine is true. Is that intended.

Reply:
set isMajorMine=true in the test which now renders the full component